build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -3,6 +3,9 @@
   <Import Project="test/ProjectsToTest.props" />
   <Import Project="test/TestPackageProjects.props" />
 
+  <UsingTask TaskName="DotNetBuild" AssemblyFile="$(CLIBuildDll)" />
```

Comment:
Would it make sense to put all these UsingTask elements in build.proj?  That way they aren't duplicated in every .targets file that calls them.  (I see Prepare already has DotNetRestore.)